webrtc.html

```diff
@@ -2336,8 +2355,67 @@
           <dd>
             <p>See <a href="#widl-RTCPeerConnectionIceEvent-candidate"><code>RTCPeerConnectionIceEvent.candidate</code></a>.</p>
           </dd>
+
+          <dt>DOMString url</dt>
+        </dl>
+      </section>
+
+      <section>
+        <h4>RTCPeerConnectionIceErrorEvent</h4>
+
+        <p>The <code>icecandidateerror</code> event of the RTCPeerConnection uses
+        the <code><a>RTCPeerConnectionIceErrorEvent</a></code> interface.</p>
+
+        <dl class="idl" data-merge="RTCPeerConnectionIceErrorEventInit" title=
+        "interface RTCPeerConnectionIceErrorEvent : Event">
+          <dt>Constructor(DOMString type, RTCPeerConnectionIceErrorEventInit
+          eventInitDict)</dt>
+
+          <dt>readonly attribute DOMString hostCandidate</dt>
+
+          <dd>
+            <p>The <code>hostCandidate</code> attribute is the local IP address and
+            port used to communicate with the STUN or TURN server.</p>
+            <p> On a multihomed system, multiple interfaces may be used to contact the
+            server, and this attribute allows the application to figure out
+            on which one the failure occurred.</p>
```

Comment:
As we discussed, if you are suppressing host candidates for some reason, this would be the string "0.0.0.0" or "::".  You should say this.